src/Amp.php

```diff
@@ -116,6 +116,15 @@ final class Amp
     const SIZER_ELEMENT             = 'i-amphtml-sizer';
     const INTRINSIC_SIZER_ELEMENT   = 'i-amphtml-intrinsic-sizer';
 
+    /**
+     * Maximum size of the CSS styles in bytes.
+     *
+     * @todo Max size is hard-coded for now until we ported over the generated spec into a reusable package.
```

Comment:
This is more about avoiding multiple sources of truth. Once we have ported the spec over, it will be easy to refer to the value within the spec, instead of within some arbitrary class constant.